src/AnmManager.cpp

```diff
@@ -60,12 +31,23 @@ struct VertexTex1DiffuseXyz
 DIFFABLE_STATIC(VertexTex1Xyzrwh, g_PrimitivesToDrawVertexBuf[4]);
 DIFFABLE_STATIC(VertexTex1DiffuseXyzrwh, g_PrimitivesToDrawNoVertexBuf[4]);
 
-AnmManager::AnmManager() : virtualMachine()
+#ifndef DIFFBUILD
```

Comment:
I assume this is necessary because DIFFBUILD_STATIC doesn't have an assign variant? I think I'd rather have a

```c
DIFFBUILD_STATIC_ASSIGN(D3DFORMAT, g_TextureFormatD3D8Mapping[6], {
    D3DFMT_UNKNOWN, D3DFMT_A8R8G8B8, D3DFMT_A1R5G5B5, D3DFMT_R5G6B5, D3DFMT_R8G8B8, D3DFMT_A4R4G4B4,
});
```

Hiding the ifdef, for consistency.

Reply:
assign on macro seems broken with our compiler...

Reply:
😢